test/e2e/clusterpropagationpolicy_test.go

```diff
@@ -49,6 +50,7 @@ var _ = ginkgo.Describe("[BasicClusterPropagation] basic cluster propagation tes
 				_, err := karmadaClient.PolicyV1alpha1().ClusterPropagationPolicies().Create(context.TODO(), crdPolicy, metav1.CreateOptions{})
 				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
 			})
+			time.Sleep(20 * time.Second)
```

Comment:
This is a little bit urgly to wait the new-installed APIs(CRD) reflected to `Cluster`.

I have an idea that modifies the `check if the crd present on member clusters` below,
we don't have to check the CRD existence at the member cluster but check the API enablement from `Cluster`?